resume.css

```
@@ -0,0 +1,80 @@
+body {
+	font-family: 'Poppins', sans-serif;
+}
+
+.header {
+	background-color: #b0d1cc;
+	text-align: center;
+}
+
+.bottom {
+	background-color: #b0d1cc;
+	text-align: center;
+	width: 100%;
+	height: 100px;
+}
+
+h1 {
+	font-size: 40px;
+	padding-top: 100px;
+	margin-top: 0px;
+	margin-bottom: 0px;
+}
+
+h2 {
+	font-size: 20px;
+	padding-bottom: 20px;
+	margin-top: 0px;
+	margin-bottom: 0px;
+}
+
+.personal-detail {
+	display: flex;
+	margin: 50px 80px 50px 80px;
```

Can use shorthand syntax
```suggestion
	margin: 50px 80px;
```